### PR TITLE
Fix error, notes, and warning based on BiocCheck()

### PR DESCRIPTION
**Pull request by erawijantari**

### CODE REVIEW

**Review by antagomir — APPROVED** [review]

File: R/splitOn.R [hunk 1]
```diff
@@ -289,7 +289,7 @@ setMethod("splitOn", signature = c(x = "TreeSummarizedExperiment"),
         # Manipulate rowTree or not?
         if( update_rowTree ){
             # If the returned value is a list, go through all of them
-            if( class(x) == "SimpleList" ){
+            if(is(x, 'SimpleList')){#( class(x) == "SimpleList" ){
```
Comment (TuomasBorman) [inline R/splitOn.R]: this is commented #( class(x) == "SimpleList" ){?

Comment (TuomasBorman) [reply R/splitOn.R]: remove the commented code

**Comment by antagomir** [comment]
fine i guess

**Comment by antagomir** [comment]
Is this ready?

**Comment by antagomir** [comment]
Is this ready? 

@erawijantari - can you resolve the conflicts, and pull this through so that we can close the PR?

**Comment by antagomir** [comment]
@erawijantari

**Comment by antagomir** [comment]
@erawijantari - can you resolve the conflicts, and pull this through so that we can close the PR?

**Comment by erawijantari** [comment]
Sorry, seems like I accidentally closed the PR when I synced the forked directory and changed the master branch to devel as was suggested.
But I had checked all change and did not see any conflict.

**Review by antagomir — APPROVED** [review]

**Comment by antagomir** [comment]
Seems good, thanks!

**Pull request merged**